File: P1S2/src/Ordenador.java
public class Ordenador {
    private String codigo;
    private String procesador;
    private String memoria;

    public Ordenador(String codigo, String procesador, String memoria) {
        this.codigo = codigo;
        this.procesador = procesador;
        this.memoria = memoria;
    }

    @Override
    public String toString() {
        return "Ordenador{" +
                "codigo='" + codigo + '\'' +
                ", procesador='" + procesador + '\'' +
                ", memoria='" + memoria + '\'' +
                '}';
    }
}

File: P1S2_Ruby/ordenador.rb
class Ordenador
  def initialize(codigo, procesador, memoria)
    @codigo = codigo
    @procesador = procesador
    @memoria = memoria
  end

  def to_s
    "Ordenador{codigo='#{@codigo}', procesador='#{@procesador}', memoria='#{@memoria}'}"
  end
end
